src/script/Cargo.toml

```diff
@@ -46,17 +46,17 @@ futures-util.workspace = true
 once_cell = "1.17.0"
 paste = { workspace = true, optional = true }
 query = { path = "../query" }
-# TODO(discord9): This is a forked and tweaked version of RustPython, please update it to newest original RustPython After Update toolchain to 1.65
-rustpython-ast = { git = "https://github.com/discord9/RustPython", optional = true, rev = "2e126345" }
-rustpython-codegen = { git = "https://github.com/discord9/RustPython", optional = true, rev = "2e126345" }
-rustpython-compiler = { git = "https://github.com/discord9/RustPython", optional = true, rev = "2e126345" }
-rustpython-compiler-core = { git = "https://github.com/discord9/RustPython", optional = true, rev = "2e126345" }
-rustpython-parser = { git = "https://github.com/discord9/RustPython", optional = true, rev = "2e126345" }
-rustpython-pylib = { git = "https://github.com/discord9/RustPython", optional = true, rev = "2e126345", features = [
+# TODO(discord9): This is a forked and tweaked version of RustPython, please update it to newest original RustPython After RustPython support GC
```

Comment:
What's the status of the GC PR in rustpython? Are u still working on it?

Reply:
Almost done, the maintainer is currently breaking GC PR into two(First a `Traverse` feature then core of GC algorithm and currently reviewing `Traverse` PR)https://github.com/RustPython/RustPython/pull/4872

> What's the status of the GC PR in rustpython? Are u still working on it?

Reply:
Cool, great news.